packages/firebase-frameworks/package.json

```
@@ -43,7 +43,7 @@
     "dist"
   ],
   "engines": {
-    "node": "^16.0.0 || ^18.0.0 || ^20.0.0 || ^22.0.0"
+    "node": "^16.0.0 || ^18.0.0 || ^20.0.0 || ^22.0.0 || ^24.0.0"
```

Just confirming we need this change as the Firebase CLI itself errors out when trying to deploy with v16/18. @service-paradis FYI created a PR here https://github.com/firebase/firebase-tools/pull/9599 to handle it in the CLI too

```suggestion
    "node": "^20.0.0 || ^22.0.0 || ^24.0.0"
```